lib/internal/repl/await.js

```diff
@@ -92,13 +97,38 @@ for (const nodeType of ObjectKeys(walk.base)) {
 }
 
 function processTopLevelAwait(src) {
-  const wrapped = `(async () => { ${src} })()`;
+  const wrapPrefix = '(async () => {\n';
```

Comment:
would be nice to avoid a newline here so error line info doesn't get messed up

Reply:
Thanks, fixed.